packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -102,6 +102,28 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
         throw new TRPCError({ code: "BAD_REQUEST", message: t("username_already_taken") });
       }
     }
+  } else if (input.username && user.organizationId && user.movedToProfileId) {
```

Comment:
I think we can simplify it by simply removing `username` from the data as username isn't allowed to be changed for an organization profile.